src/components/sidebar/routesView/routeShow.scss

```diff
@@ -2,11 +2,32 @@
 @import '../../shared/styles/flexbox';
 
 .routeShowView {
+    .groupedRoutes {
+        margin-left: -10px;
+        margin-right: -10px;
+        padding: 10px 10px;
```

Comment:
move padding from _RouteList_ to _RouteShowView_ to simplify and get rid of these - margins